container/openejb-core/pom.xml

```diff
@@ -580,6 +580,22 @@
       <groupId>org.apache.xbean</groupId>
       <artifactId>xbean-bundleutils</artifactId>
     </dependency>
+    <dependency>
```

Comment:
open point: should it go in a new openejb-junit5 module? Having testing stack in openejb-core is an inheritance but bothering at some point so maybe time to clean it up? wdyt?

Reply:
I think it would be (a) cleaner and (b) could allow user's to exclude the compile dependencies towards junit4  - so +1 for `openejb-junit5`. 

Could also include the classes from `openejb-junit5-backward`, so we have all junit 5 related things in one module. wdyt?

Reply:
Just moved everything into `openejb-junit5`. 

Should we also include the contents of `openejb-junit5-backward` in this new  `openejb-junit5` module? We haven't released it yet, so it wouldn't bother anyone.

Would bring a dependency towards the legacy testing stuff:

```
 <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>openejb-junit</artifactId>
            <version>${project.version}</version>
        </dependency>
```

Might be cleaner without it? wdyt @rmannibucau ?

Reply:
1. +1 -> I agree.
2. Yes. It is a totally different approach, so keeping them separated is fine for me. Users may consume them as needed.